packages/ripple-nuxt-tide/lib/middleware/page-head.js

```diff
@@ -30,7 +30,8 @@ const tidePageHead = (context, pageData) => {
       siteSectionName: siteSection ? siteSection.name : '',
       image,
       imageAlt,
-      pageType: pageData.tidePage.type
+      pageType: pageData.tidePage.type,
+      robotsNoIndex: pageData.tidePage.appMetatag.robots
     }
 
     context.store.dispatch('tide/setPageHead', getPageHeadConfig(headData))
```

Comment:
I can see this is being used in `tide.middleware.js`, do we need to update it in there or it's not used?

Reply:
Hey, @rubyllamadora I think for all the page header we are capturing the data here and if we want to handle these values differently for a pagetype then we do that in `tide.middleware.js`